services/mpa/client/client.go

```diff
@@ -103,7 +104,9 @@ func getAction(ctx context.Context, state *util.ExecuteState, c pb.MpaClientProx
 	return anyAction
 }
 
-type approveCmd struct{}
+type approveCmd struct {
+	skipConfirmation bool
+}
 
 func (*approveCmd) Name() string     { return "approve" }
 func (*approveCmd) Synopsis() string { return "Approves an MPA request" }
```

Comment:
Could you mention the confirmation in the usage?

Reply:
Added